src/wren/Renderable.hpp

```diff
@@ -98,7 +98,7 @@ namespace wren {
     Renderable();
     virtual ~Renderable();
 
-    void setupAndRender(const ShaderProgram *program);
+    void setupAndRender(ShaderProgram *program);
```

Comment:
Why did you remove the const qualifiers from these methods?

Reply:
I needed to remove them for another way to fix this issue, but I can re add these `const` with the current implementation.